site.json

```diff
@@ -34,5 +34,13 @@
       "link": "https://nodejs.org/en/blog/vulnerability/february-2024-security-releases/"
     }
   },
-  "websiteBadges": {}
+  "websiteBadges": {
+    "index": {
+      "startDate": "2024-03-24T13:34:50.449Z",
+      "endDate": "2024-03-31T13:34:50.449Z",
+      "kind": "default",
+      "title": "Diving into the Node.js Website Redesign",
```

Comment:
```suggestion
      "title": "New Blog Post",
      "text": "Diving into the Node.js Website Redesign",
```

if you do this change it's will look like that (better)

<img width="1436" alt="Capture d’écran 2024-03-24 à 18 45 49" src="https://github.com/nodejs/nodejs.org/assets/97875033/1aff34bd-4bd6-42e6-9608-10930cd34bb6">